build.sbt

```diff
@@ -187,6 +187,26 @@ lazy val `java-eventing-shopping-cart` = project
   )
   .settings(attachProtobufDescriptorSets)
 
+lazy val `java-car-metrics` = project
+  .in(file("samples/java-car-metrics"))
+  .dependsOn(sdk, testkit % IntegrationTest)
+  .enablePlugins(AkkaGrpcPlugin, IntegrationTests, LocalDockerImage)
+  .settings(
+    name := "java-metrics-car",
+    Compile / mainClass := Some("metrics.Main"),
+    Compile / akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Java),
+    Compile / javacOptions ++= Seq("-encoding", "UTF-8", "-source", "11", "-target", "11"),
+    libraryDependencies ++= Seq(
+        "ch.qos.logback" % "logback-classic" % LogbackVersion,
+        "com.novocode" % "junit-interface" % JUnitInterfaceVersion % IntegrationTest
+      ),
+    testOptions += Tests.Argument(TestFrameworks.JUnit, "-q", "-v", "-a"),
+    IntegrationTest / akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Java),
+    IntegrationTest / akkaGrpcGeneratedSources := Seq(AkkaGrpc.Client),
+    IntegrationTest / PB.protoSources ++= (Compile / PB.protoSources).value
+  )
+  .settings(attachProtobufDescriptorSets)
```

Comment:
I'm wondering if there is any sbt plugin that we can use to delegate to mvn. 

This setup will work for simple projects, but we may run into issue when we start to deal dependencies in the project. We don't want to maintain deps here and in the maven. 

We may also run into other build things that we configure in the pom.xml and then again here. The gRPC setup is one of them, but we may have others.

Reply:
I think it is aligned with the other projects @octonato. I compared it to `java-eventing-shopping-cart` and seems identical to me.  Maybe I'm not understanding exactly what do you refer to with codegen.

Reply:
Yep, it's aligned with the current status quo and that's the problem. 

We want all samples and docs to be based to the codegen. Have a look at https://github.com/lightbend/akkaserverless-java-sdk/pull/10 and https://github.com/lightbend/akkaserverless-java-sdk/pull/12

In the meantime, after many try and error and fighting with CircleCI, I managed to integrate the maven build for those two projects in the CI.

Reply:
done now @octonato

Reply:
@franciscolopezsancho, maybe you should rebase your PR with #10 and add a use the trick I added to build the maven with the recently built SDK version.

Reply:
Then better to send your PR targeting the branch from #10, ie: https://github.com/lightbend/akkaserverless-java-sdk/tree/rgc/codegen-value-entity-sample.

We can then do cascade merging once we start to get them in `master`